components/footer.css

```diff
@@ -0,0 +1,35 @@
+.footer {
+    background-color: #956cf3;
+    padding: 2rem 1rem;
+    text-align: center;
+    color: white;
+    border-top-right-radius: 1rem;
+}
+.footer .footer-header {
+    font-weight: 700;
+    font-size: 3rem;
+}
+
+.footer .footer-link{
+    color: white;
+    font-size: 2rem;
+}
+
+.footer ul{
+    padding-inline-start: 0px;
+}
+
+.list-non-bullet {
+    list-style: none;
+}
+
+.list-item-inline {
+    display: inline;
+    padding: 0.5rem 0.5rem;
```

Comment:
Do not use rem for smaller size requirement

Reply:
ok